makerdao/governance-portal-v2
# Chore/ch365/update dark mode colors for alerts

### Link to Clubhouse story
https://app.clubhouse.io/dux-makerdao/story/365/update-dark-mode-colors-for-alerts

### What does this PR do?
Updates colors for alerts and cookie section for dark mode

### Steps for testing:
Pull branch, update alert variants on account page to see the different colors, make sure it matches expected per Figma

### Screenshots (if relevant):
Before:
<img width="1327" alt="Screen Shot 2021-08-09 at 2 35 44 PM" src="https://user-images.githubusercontent.com/5225766/128708195-cbf78ee4-bc5b-443b-a5f5-1de166fa633a.png">
<img width="1326" alt="Screen Shot 2021-08-09 at 2 36 07 PM" src="https://user-images.githubusercontent.com/5225766/128708197-d03fd388-2e31-499c-964c-d18318ebc139.png">

After:
<img width="1324" alt="Screen Shot 2021-08-09 at 2 36 48 PM" src="https://user-images.githubusercontent.com/5225766/128708211-a131f7e3-f711-4f5c-83f4-1519152e4005.png">
<img width="1328" alt="Screen Shot 2021-08-09 at 2 36 31 PM" src="https://user-images.githubusercontent.com/5225766/128708217-8bb72260-6784-4b54-a855-da236efbb17c.png">

### Add a GIF:
![](https://media.giphy.com/media/4Z3URvjmDiZF0VHWcU/giphy.gif)
